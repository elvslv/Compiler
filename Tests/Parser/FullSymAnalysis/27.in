var i, j, n, l, k: integer;
arr: array[1..10] of integer;
const m = 18;
begin
	for arr[1] := 1 to m do
		l := l + k;
end.