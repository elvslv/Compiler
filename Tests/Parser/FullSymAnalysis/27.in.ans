var I : INTEGER;
var J : INTEGER;
var N : INTEGER;
var L : INTEGER;
var K : INTEGER;
var ARR : array [const 1..const 10] of INTEGER;
const M = 18;

9 5 Illegal counter variable