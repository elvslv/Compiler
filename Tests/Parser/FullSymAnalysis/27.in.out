var ARR : array [const 1..const 10] of INTEGER;
var I : INTEGER;
var J : INTEGER;
var K : INTEGER;
var L : INTEGER;
const M = 18;
var N : INTEGER;

9 5 Illegal counter variable